lib/services/azuresqldb/cmd-bind.js

```diff
@@ -25,6 +27,14 @@ var sqldbBind = function (params) {
                 databaseLoginPassword: reqParams.userProvidedDatabaseLoginPassword
             });
         }
+        var primaryAdministratorLogin, primaryAdministratorLoginPassword;
```

Comment:
I think we don't need to put "primary" as a prefix as this module is for single SQL DB. Same as other occurrences.

Reply:
Sounds good, removed "primary" from occurences in cmd-bind and cmd-unbind files.

Reply:
test-matrix2 is running lifecycle without the ability to create SQL servers, which is different from test-matrix, right? When i ran the integration tests locally, I commented out execution of test-matrix2 from lifecycle.js. If you mean that preprovision is similar to having the SQL server created manually before the tests - then yes, test-matrix2 is redundant.

Reply:
Yes, I think preProvision created server is not much different from manually created server. Could you help remove them?

Reply:
I have deleted the 'test-matrix2.js' file - it can be found in the latest commit.
https://github.com/Azure/meta-azure-service-broker/pull/211/commits/3c19c65245d5139030be9070a1f12ac39fd2f50d
Hope i'm not misunderstanding what is being looked for here.